aicsimageio/readers/czi_reader.py

```diff
@@ -0,0 +1,784 @@
+                xml=meta,
+                scene_index=self.current_scene_index,
+                image_short_info=dims_shape,
+            )
+
+            # Store pixel sizes
+            self._px_sizes = px_sizes
+
+            return xr.DataArray(
+                image_data,
+                dims=dims,
+                coords=coords,  # type: ignore
+                attrs={constants.METADATA_UNPROCESSED: meta},
+            )
+
+    @staticmethod
+    def _dims_list_to_dict(list_in: List) -> Dict:
+        return {x[0]: x[1] for x in list_in}
+
+    @staticmethod
+    def _dims_shape_to_scene_dims_shape(
+        dims_shape: List[Dict], scene_index: int, consistent: bool
+    ) -> Dict:
+        dims_shape_index = 0 if consistent else scene_index
+        dims_shape_dict = dims_shape[dims_shape_index]
+        dims_shape_dict.pop("S", None)
+        return dims_shape_dict
+
+    @staticmethod
+    def _stitch_tiles(
+        data: types.ArrayLike,
+        data_dims_shape: Dict,
+        bboxes: Dict,
+        mosaic_bbox: BBox,
+    ) -> types.ArrayLike:
+        # Assumptions: 1) docs for ZEISSRAW(CZI) say:
+        #   Scene – for clustering items in X/Y direction (data belonging to
+        #   contiguous regions of interests in a mosaic image).
+
+        # 'S' in czi is Scene not Samples, 'A' is sAmples
+
+        # Create empty array
+        arr_shape_list = []
+
+        ordered_dims_present = [
+            dim for dim in DEFAULT_CZI_DIMENSION_ORDER_LIST if dim in data_dims_shape
+        ]
+        for dim in ordered_dims_present:
+            if dim not in REQUIRED_CZI_CHUNK_DIMS:
+                arr_shape_list.append(data_dims_shape[dim][1])
+            if dim is DimensionNames.SpatialY:
+                arr_shape_list.append(mosaic_bbox.h)
+            if dim is DimensionNames.SpatialX:
+                arr_shape_list.append(mosaic_bbox.w)
+            if dim == "A":
+                arr_shape_list.append(data_dims_shape[dim][1])
+
+        ans = None
+        if type(data) is da.Array:
+            ans = da.zeros(
+                arr_shape_list,
+                chunks=data.chunks,  # type: ignore
+                dtype=data.dtype,
+            )
+        else:
+            ans = np.zeros(arr_shape_list, dtype=data.dtype)
+
+        for (tile_info, box) in bboxes.items():
+            # construct data indexes to use
+            tile_dims = tile_info.dimension_coordinates
+            tile_dims.pop("S", None)
+            tile_dims.pop("B", None)
+            data_indexes = [
+                tile_dims[t_dim]
+                for t_dim in DEFAULT_CZI_DIMENSION_ORDER_LIST_WITH_MOSAIC_TILES
+                if t_dim in tile_dims.keys() and t_dim not in REQUIRED_CZI_CHUNK_DIMS
+            ]
+            # add Y and X
+            data_indexes.append(slice(None))  # Y ":"
+            data_indexes.append(slice(None))  # X ":"
+            # if "A" in tile_dims.keys():
+            #     data_indexes.append(slice(None))
```

Comment:
? Shouldn't we include samples?

Reply:
Thanks for catching this. I kept looking for it but missed it. It would be great to get a Mosaic BGR from Chris for testing.